cms/wsgi.py

```diff
@@ -1,21 +1,25 @@
 # Patch the xml libs before anything else.
-from safe_lxml import defuse_xml_libs
-defuse_xml_libs()
+import os
 
 # Disable PyContract contract checking when running as a webserver
 import contracts
-contracts.disable_all()
```

Comment:
@andy-armstrong , I assume the ordering of inputs doesn't matter here, but is there a way to verify?